src/wasm.rs

```diff
@@ -753,6 +756,9 @@ impl<'a> From<&'a Operator<'a>> for Instruction {
                 offset: memarg.offset,
             },
 
+            Operator::MemorySize { .. } => Instruction::MemorySize,
+            Operator::MemoryGrow { .. } => Instruction::MemoryGrow,
```

Comment:
Are there no useful parameters to pipe down?

Reply:
Nope, the Wasm spec has a byte reserved for "memory index" which is currently unused
https://webassembly.github.io/spec/core/binary/instructions.html#memory-instructions

> Note
>
> In future versions of WebAssembly, the additional zero bytes occurring in the encoding of the memory.size
> and memory.grow instructions may be used to index additional memories.

You could pipe down this byte, but then the unused variable would move into more places in the code base.